src/api.cpp

```diff
@@ -904,6 +905,18 @@ cl_int CLVK_API_CALL clGetDeviceInfo(cl_device_id dev,
         copy_ptr = &val_pci_bus_info;
         size_ret = sizeof(val_pci_bus_info);
         break;
+    case CL_DEVICE_SUB_GROUP_SIZES_INTEL:
+        if (device->supports_controled_subgroups()) {
```

Comment:
```suggestion
        if (device->supports_subgroup_size_selection()) {
```
Or maybe `supports_subgroup_size_control` so it's a bit more explicit than "controled_subgroups".